paper-api/src/main/java/org/bukkit/entity/Creaking.java

```diff
@@ -1,8 +1,45 @@
 package org.bukkit.entity;
 
+import org.bukkit.Location;
+import org.jspecify.annotations.NullMarked;
+
 /**
  * Represents a Creaking.
  */
+@NullMarked
 public interface Creaking extends Monster {
 
+    /**
+     * Gets the home location for this Creaking (aka where can be a {@link org.bukkit.block.CreakingHeart}).
+     *
+     * @return the location of the home.
+     */
+    Location getHome();
+
+    /**
+     * Sets the home location for this Creaking.
+     *
+     * @param location the location of the home.
+     */
+    void setHome(final Location location);
+
+    /**
+     * Activate this Creaking to target and follow a player.
+     *
+     * @param player the target.
+     */
+    void activate(final Player player);
+
+    /**
+     * Deactivate this Creaking to the current target player.
```

Comment:
`Deactivates`

Reply:
Also, phrasing feels off here too.

Reply:
maybe can specify the behaviour better here...